assets/js/editor-components/sidebar-compatibility-notice/use-compatibility-notice.ts

```diff
@@ -10,7 +10,7 @@ export const useCompatibilityNotice = (
 	blockName: string
 ): [ boolean, () => void ] => {
 	const [ dismissedNotices, setDismissedNotices ] = useLocalStorageState(
-		`wc-blocks_dismissed_compatibility_notices`,
+		`wc-blocks_dismissed_sidebar_compatibility_notices`,
```

Comment:
Done so merchants who have dismissed the current notices will see the new one.

Reply:
Dismissing the notice for Checkout doesn't dismiss it for Cart, should we unify both?